sklearn/utils/_mocking.py

```diff
@@ -115,6 +115,20 @@ def predict(self, T):
             assert self.check_X(T)
         return self.classes_[np.zeros(_num_samples(T), dtype=np.int)]
 
+    def predict_proba(self, T):
```

Comment:
why `T`? The scikit-learn convention is `X` for the input data of a statistical estimator.

Reply:
I tried to stick close to the `predict` method, which also uses `T`. But I can change it (as well as `predict`).

Reply:
please do change it.